components/FormStyles/InputBox.tsx

```diff
@@ -1,21 +1,28 @@
 import styled from 'styled-components'
 import { FieldHookConfig, useField } from 'formik'
 import React, { ChangeEvent } from 'react'
+import { COLORS } from '../../constants/colors'
 
 interface InputBoxProps {
-  width: string
-  height: string
+  width: number
```

Comment:
most of this file is just converting the width/height to be a string aside from the syling in the inputBox below